Readme.md

```diff
@@ -4,21 +4,24 @@ https-redirect
 
 ## Usage
 
-This will set up an http server on port `8080` which redirects all incoming
-requests to the corresponding https url. The `-1` flags will make the http
-server send `301` status code.
-
-    $ https-redirect -p 8080 -1
+This will set up an http server on port `8080` which redirects all incoming requests to the corresponding https url. The `-1` flags will make the http server send `301` status code.
```

Comment:
> Sorry, it is my personal coding style I'm used to.

No problem. Here is the style I'm using: https://github.com/bengourley/js-style-guide